dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -160,6 +161,19 @@ public void findAllRelationshipTest() throws Exception {
                        RelationshipMatcher.matchRelationship(relationship3)
                    )))
         ;
+
+        getClient().perform(get("/api/core/relationships").param("size", "2"))
```

Comment:
I would prefer a separate test method for the pagination as usually done but this is not a stopping issue of course